src/Interface/Modules/Render/Screenshot.h

```diff
@@ -46,13 +46,13 @@ namespace SCIRun
     public:
       explicit Screenshot(QGLWidget *glwidget, QObject *parent = nullptr);
```

Comment:
This won't merge easily with https://github.com/SCIInstitute/SCIRun/pull/1979/files#diff-09b4cb29ce5bcc63aba5130a741d0f1a. I guess we can wait and see after #1979 is merged into qt5-dev.

Reply:
I think the function for grabbing frame buffer is still the same.